rustls/src/server/server_conn.rs

```diff
@@ -154,6 +154,20 @@ impl<'a> ClientHello<'a> {
     /// Get the alpn.
     ///
     /// Returns `None` if the client did not include an ALPN extension
+    ///
+    /// ALPN (Application Layer Protocol Negotiation) contains a set of identifiers
```

Comment:
I'd rewrite this more like this: "Application Layer Protocol Negotiation (ALPN) is a TLS extension that lets a client submit a set of identifiers that each a represent an application-layer protocol. The server will then pick its preferred protocol from the set submitted by the client. Each identifier is represented as a byte array, although common values are often ASCII-encoded."

And probably link to the RFC: https://datatracker.ietf.org/doc/html/rfc7301.